src/annotator/guest.js

```diff
@@ -658,9 +652,7 @@ export default class Guest extends Delegator {
     }
 
     this.selectedRanges = [range];
-    if (this.toolbar) {
-      this.toolbar.newAnnotationType = 'annotation';
-    }
+    this.publish('hasSelectionChanged', [true]);
 
```

Comment:
Similar to `highlightsVisibleChanged`:

```suggestion
    this.publish('selectionChanged', [true]);
    
```

Reply:
Hmm - I debated this. In the end I chose `hasSelectionChanged` rather than `selectionChanged` to make it more obvious that the argument was a boolean (whether there is a selection) rather than the current selection itself.